activerecord/lib/active_record/connection_adapters/abstract/schema_statements.rb

```diff
@@ -1142,6 +1142,9 @@ def check_constraints(table_name)
       #
       #   ALTER TABLE "products" ADD CONSTRAINT price_check CHECK (price > 0)
       #
+      # The +options+ hash can include the following keys:
```

Comment:
The example above uses the `:name` option, would you mind documenting that one here too?